api/apps/api/test/proxy.vector-tiles.e2e-spec.ts

```diff
@@ -109,78 +118,98 @@ describe('ProxyVectorTilesModule (e2e)', () => {
           .expect(HttpStatus.OK);
       });
       describe('Filter by guid', () => {
-        test.skip('guid country level', async () => {
+        test('guid country level', async () => {
+          const response = await request(app.getHttpServer())
+            .get('/api/v1/administrative-areas/0/preview/tiles/6/30/25.mvt')
+            .set('Authorization', `Bearer ${jwtToken}`)
+            .expect(HttpStatus.OK);
+        });
+        test('guid adm1 level', async () => {
           const response = await request(app.getHttpServer())
-            .get('/api/v1/administrative-areas/1/preview/tiles/100/60/30.mvt')
+            .get('/api/v1/administrative-areas/1/preview/tiles/6/30/25.mvt')
             .set('Authorization', `Bearer ${jwtToken}`)
             .expect(HttpStatus.OK);
         });
-        test.skip('guid adm1 level', async () => {
+        test('guid adm2 level', async () => {
           const response = await request(app.getHttpServer())
-            .get('/api/v1/administrative-areas/1/preview/tiles/100/60/30.mvt')
+            .get('/api/v1/administrative-areas/2/preview/tiles/6/30/25.mvt')
             .set('Authorization', `Bearer ${jwtToken}`)
             .expect(HttpStatus.OK);
         });
+        test('guid level > 2 should not be allowed', async () => {
+          const response = await request(app.getHttpServer())
+            .get('/api/v1/administrative-areas/3/preview/tiles/6/30/25.mvt')
+            .set('Authorization', `Bearer ${jwtToken}`)
+            .expect(HttpStatus.BAD_REQUEST);
+        });
       });
-
-      test.skip('Filter by bbox', async () => {
+      test('Filter by bbox', async () => {
         const response = await request(app.getHttpServer())
-          .get('/api/v1/administrative-areas/1/preview/tiles/100/60/30.mvt')
+          .get('/api/v1/administrative-areas/1/preview/tiles/6/30/25.mvt?bbox=[10,10,5,5]')
           .set('Authorization', `Bearer ${jwtToken}`)
           .expect(HttpStatus.OK);
       });
-
-      test('Should simulate an error if input is invalid', async () => {
+      test('Should throw a 400 error if filtering by z level greater than 20', async () => {
         const response = await request(app.getHttpServer())
-          .get('/api/v1/administrative-areas/1/preview/tiles/100/60/30.mvt')
+          .get('/api/v1/administrative-areas/0/preview/tiles/21/30/25.mvt')
           .set('Authorization', `Bearer ${jwtToken}`)
           .expect(HttpStatus.BAD_REQUEST);
       });
-
-      test('Should throw a 400 error if filtering by level other than 0, 1 or 2', async () => {
+    });
+    describe('WDPA preview vector tiles.', () => {
```

Comment:
not for this PR, but I think it's worth splitting up these tests according to the kind of tiles we test, similarly to what we did for the actual controllers.

Reply:
Well indeed the testing is very similar, check that the tiles are being generated as expected. If this files grows a lot i will do it ofc